# should be number_input or some other valid function: @test err import
from pycasio.casio.input import notafunction
